packages/pakistan/README.md

```diff
@@ -0,0 +1,170 @@
+# 🇵🇰 Paksitan
+
+Contains all the districts of Pakistan:
+
+* Abbottabad
+* Astore
+* Attock
+* Awaran
+* Badin
+* Bagh
+* Bahawalnagar
+* Bahawalpur
+* Bajaur
+* Bannu
+* Barkhan
+* Battagram
+* Bhakkar
+* Bhimber
+* Bolan
+* Buner
+* Chaghai
+* Chakwal
+* Charsadda
+* Chiniot
+* Chitral
+* Dadu
+* Dera Bugti
+* Dera Ghazi Khan
+* Dera Ismail Khan
+* Diamer
+* Bannu District
+* D I Khan District
+* Kohat District
+* Lakki District
+* Peshawar District
+* Tank District
```

Comment:
Why are there duplicated districts (Bannu and Bannu District, Tank and Tanks District...)? 🤔

Reply:
Confusing stuff, these new districts were named after the old ones with addition to "FR -> Frontier Region in their names" now its changed to Sub-Division. So Bannu and Bannu Subdivision are two different places. :/

Reply:
I have added the "subdivision" so one can differentiate which district it is.

Reply:
Okay, thanks for the explanation 👌